package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "lint": "jscs `find ./lib -name '*.js'` -v && jscs `find ./test -name '*.js'` -v",
     "test": "npm run unit-test",
-    "test-windows": "tap -R spec test/*.test.js --timeout=120",
+    "test-windows": "tap -R spec test/*.test.js --timeout=1200",
```

Comment:
do we really need 20 mins?

Reply:
yes, on windows (appveyor) the tests running powershell takes 5-10 minutes. I put 20 just in-case. Running test on VM with win 10 took 3 minutes for that test (which would have failed in 2 minutes timeout).

Reply:
Only for a single dll check? Is this what's taking so long?